### ISSUE TITLE
How to link marginalia to the main textq

### ISSUE THREAD

**Issue by seumasjeltzz** [seumasjeltzz/LinguaeGraecaePerSeIllustrata#14]
Just wanted to open up a general discussion about best ways to tie-in margin-type notes: graphics, short explanations, alternative renderings, etc., that in the original LLPSI lie in the margins, but here would I suppose be tied to specific words or multi-word expressions. Thoughts welcome.

**Comment by jtauber** [comment]
As the structure of the information becomes any more complex than basic headings, paragraphs, and lists, I think we need to consider switching from markdown to XML.

**Comment by seumasjeltzz** [comment]
Right. I wonder then if a first step isn't to mirror the /src markdown files and start converting them one by one to XML, while I also learn XML at the same time.

**Comment by jtauber** [comment]
Yeah, I think mirroring is the way to go. Learning XML isn't the issue so much as deciding what particular vocabulary to use. I would suggest you may as well adopt a lightweight subset of TEI (as that will be worth learning anyway)

The nice thing is that will give us a lot more control both in generated nicely formatted versions for display and also extracting necessary data for linguistic analysis.

**Comment by marktaber** [comment]
Is there a preferred/standard toolset for TEI? I've been out to the TEI website (SGML rises again! It never died! :-D) and many of the tools look, if not abandoned, then not "fresh." (Tumbleweeds). So what tools are being used for creating TEI documents in the 21st century?


Thanks much,
Mark


Mark Taber
"To know that one doesn't know is best." - Laozi

On April 30, 2019 at 3:15 PM, James Tauber <notifications@github.com> wrote:



Yeah, I think mirroring is the way to go. Learning XML isn't the issue so much as deciding what particular vocabulary to use. I would suggest you may as well adopt a lightweight subset of TEI (as that will be worth learning anyway)

The nice thing is that will give us a lot more control both in generated nicely formatted versions for display and also extracting necessary data for linguistic analysis.

**Comment by honza** [comment]
This is the first time I have heard of TEI. It seems interesting but also kind of overwhelming. Is there an example project somewhere that we'd want to emulate? What are some recommended resources for learning this beast?

**Comment by ldutra** [comment]
I seem to remember there were some more approchable tools in the Crosswire Bible Society.  Do not remember if they were directly related to TEI.  Not sure they would be enough too.

**Comment by ryanfb** [comment]
In the short term, this might have some templates or ideas that would be useful for achieving marginal notes with Pandoc: https://github.com/duzyn/tufte-markdown

**Comment by jtauber** [comment]
@honza TEI is only as complex as you want/need it to be. At one level, it's not really any more complex than HTML until you need it to be.

My main was there's no point inventing a vocabulary if TEI has already done the work.

Suitable markdown extensions may suffice. I'm imagining the linguistic annotation I'm doing will largely stay standoff anyway.
